operations/healthcheck.md

```diff
@@ -0,0 +1,34 @@
+<!-- AUTOMATICALLY GENERATED, DO NOT MODIFY -->
+# HealthCheck
+
+## Get health check status
+
+Returns a health check object with status message.
+
+### Request
+
+`[PlatformAddress]/api/v2/health-check`
```

Comment:
Should that be `/api/pos/v1`?

Reply:
@MikeAdamsMews Good question. Please allow me to add some context: we already have a `/v1` api which is mostly used internally. This is meant as a new public api hence it's `/v2`. Apart from that, we are still following the naming conventions of `/v1`.